src/content/docs/agents/net-agent/getting-started/8x-to-9x-agent-migration-guide.mdx

```diff
@@ -1,38 +1,40 @@
+In 9.x, this will be reversed: `crossApplicationTracer` will not be present by default, and `distributedTracing` will be present with the default value of `enabled=”true”`.  However, the agent installers do not overwrite an existing `newrelic.config` file when upgrading from one version to another. Therefore, if you are upgrading the agent from 8.x to 9.x on a given host, the agent’s behavior on that host will not change.
+
+<Callout variant="tip">
+  If you wish to adopt the new default tracing behavior when upgrading from 8.x to 9.x, you will need to modify your agent configuration to enable distributed tracing. New installations of the 9.x agent on a host will have distributed tracing enabled by default.
+</Callout>
 
 
 ## Removal of deprecated public agent API methods
 
-All deprecated API methods have replacement API methods with equivalent functionality.  If you are using the .NET agent API in your application code, we recommend that you update the package reference in your project to the latest 9.x version of the agent API assembly before upgrading the .NET agent to 9.x.  That way, you will get compile-time errors if your code is using any of the API methods removed in 9.x.  If you continue to use an 8.x or earlier version of the API assembly, and your code uses any of the deprecated APIs listed below, you will not get compile-time errors, but you will get run-time errors if you instrument your application with a 9.x agent version, and the API methods will have no effect.
+All deprecated API methods have replacement API methods with equivalent functionality.  If you are using the .NET agent API in your application code, we recommend that you update the package reference in your project to the latest 9.x version of the agent API assembly before upgrading the .NET agent to 9.x. That way, you will get compile-time errors if your code is using any of the API methods removed in 9.x.
+
+* **8.x versions:** If you continue to use an 8.x or earlier version of the API assembly, and your code uses any of the deprecated APIs listed below, you will not get compile-time errors.
```

Comment:
I think the way this has been reworded is misleading, because of the distinction between the version of the agent API assembly referenced in a customer's code, versus the version of the agent being used to instrument the application created by compilation of that code.  These two versions are independent; it is possible for a customer to use an 8.x version of the API and use a 9.x version of the agent to instrument the app.

Here's the complete "two-by-two matrix" of possible combinations and what the user would experience when using any of the deprecated API methods being removed in 9.x in that case:

8.x API assembly, 8.x agent: no problem!  These methods aren't removed yet in the 8.x agent.
8.x API assembly, 9.x agent: the customer's app code will still compile, but the API methods will no longer have their intended effect, and runtime errors will be seen in the agent log.
9.x API assembly, 8.x agent: compile-time errors will prevent the customer's app from being built, so the agent behavior doesn't really matter.
9.x API assembly, 9.x agent: ditto.

I don't think we necessarily need to make this matrix explicit in the docs.  We're mostly concerned about the second scenario, where if the customer doesn't update their API assembly version to 9.x in their code before upgrading their agent to 9.x, they probably won't notice that their API methods are no longer having any effect.  We want them to end up in either scenario 3 or 4 so they notice the problem (because their app won't compile).  Maybe we should do a tip callout on the "we recommend..." part.

Reply:
Per slack, we've agreed on a color-coded solution. I'll make the change now. Thanks for pointing this out!